pkg/runtime/types.go

```diff
@@ -80,17 +86,19 @@ type TypeMeta struct {
 //
 // +protobuf=true
 type RawExtension struct {
-	// RawJSON is the underlying serialization of this object.
-	RawJSON []byte
+	// Raw is the underlying serialization of this object.
```

Comment:
So adding ContentEncoding / Type is interesting - when this object is included in an external type for JSON, it's assumed to be JSON (and there is no way to set encoding or content type).  For Protobuf, it will be slightly different (the nested object would be a runtime.Unknown, but ContentEncoding and ContentType would be inside that object, not this one).  For serializing nested objects (like List) we've said that that has to be something that serializer knows how to do, so the serializer is responsible for transforming Object -> Raw and vice versa.  

Do we *need* to add them here?  Maybe I didn't catch why they were required.